lib/developer_portal/app/views/developer_portal/password/new.html.liquid

```diff
@@ -6,7 +6,7 @@
         Forgot password
       </div>
       <div class="panel-body">
-        {% form 'password_reset', { class: 'form-horizontal' } %}
+        {% form 'password_reset', account, class: 'form-horizontal' %}
```

Comment:
@hallelujah
Could I use `current_account` here directly?

Reply:
You do not need to use that here, but in the ruby file  `form`

Reply:
Especially if you change it here, then you will need a migration for all the current customers (because this liquid file is stored in the database)
Which is something really to avoid as most as possible